astropy/units/quantity_helper/function_helpers.py

```diff
@@ -1042,3 +1055,54 @@ def eig(a, *args, **kwargs):
     from astropy.units import dimensionless_unscaled
 
     return (a.value,)+args, kwargs, (a.unit, dimensionless_unscaled), None
+
+
+@function_helper(module=np.lib.recfunctions)
+def structured_to_unstructured(arr, *args, **kwargs):
+    """
+    Convert a structured quantity to an unstructured one.
+    This only works if all the units are compatible.
```

Comment:
Add a line break for later typesetting (see other comment)

Reply:
✅